allure-testng/src/main/java/io/qameta/allure/testng/config/AllureTestNgConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.qameta.allure.testng.config;
+
+import io.qameta.allure.util.PropertiesUtils;
+
+import java.util.Properties;
+
+public class AllureTestNgConfig {
+
+    public static final String ALLURE_TESTNG_CONFIG = "allure.testng.config.";
+    public static final String ALLURE_REPORT_DISABLED_TESTS = ALLURE_TESTNG_CONFIG + "report.disabled.tests";
+    private boolean disabledTestsReported;
+
+    public AllureTestNgConfig() {
+        final Properties properties = PropertiesUtils.loadAllureProperties();
```

Comment:
lets keep this POJO as simple as possible. Move initialisation to static factory method

Reply:
Done, pls check.